src/main/java/frc/robot/Robot.java

```diff
@@ -10,6 +10,7 @@
 import org.littletonrobotics.junction.LoggedRobot;
 
 public class Robot extends LoggedRobot {
+  public static final boolean isCompetition = false;
```

Comment:
How is this going to be set? Do you intend to change this value and deploy at competition? That's a recipe for last-minute bugs.

Reply:
Yep, that is how we have been doing it in the main robot.  There are certainly other ways to approach this.
( File in deploy.  Preferences maybe. )